samples/tfm/tfm_secure_peripheral/README.rst

```diff
@@ -0,0 +1,166 @@
+.. _tfm_secure_peripheral_partition:
+
+TF-M secure peripheral partition
+#######################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The TF-M secure peripheral partition sample demonstrates the configuration and usage of secure peripherals in a Trusted Firmware-M (TF-M) partition.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf5340dk_nrf5340_cpuapp_ns, nrf9160dk_nrf9160_ns
+
+The sample also requires a logic analyzer.
+
+Overview
+********
+
+A secure partition is an isolated module that resides in TF-M.
+It exposes a number of functions or secure services to other partitions and/or to the non-secure firmware.
+TF-M already contains standard partitions such as crypto, protected storage and firmware update, but you can also create your own partitions.
+
+The sample demonstrates how to configure peripherals as secure peripherals and use them in the secure partition.
+In this way, the peripheral is only accessible from the secure domain, or from the secure partition with isolation level 2 or higher.
+
+The secure partition is located in the ``secure_peripheral_partition`` directory.
+It contains the partition sources, build files and build configuration files.
+The partition is built by the TF-M build system.
+See :ref:`tfm_build_system` for more details.
+
+For more information on how to add custom secure partitions refer to `TF-M secure partition integration guide`_.
+
+Configuration
+*************
+
+In order to configure a peripheral as secure, the peripheral must be enabled and assigned to the partition, and any interrupt signals must be defined.
+
```

Comment:
```suggestion

|config|

```
We always use the stock phrase that is included with |config| in the Configuration section.

This adds the following text:
See [Configuring your application](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_modifying.html#configure-application) for information about how to permanently or temporarily change the configuration.